libs/platform/ebpf_error.c

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#if !defined(_AMD64_)
+#define _AMD64_
+#endif
+#include <ntdef.h>
+#include <ntstatus.h>
+#include <ebpf_result.h>
+
+NTSTATUS
```

Comment:
Add a comment referencing the other places to be consistent with, as appeared in the location this code was copied from.  If you want, you can also add a TODO referencing issue #595 since this PR does do part of that issue, which is great.